package.json

```
@@ -17,6 +17,7 @@
     "start:dynamic": "lerna run dev --scope dynamic-pages --stream",
     "start:common": "lerna run start --scope common --stream",
```

```suggestion
    "start:common": "lerna run start --scope @codesandbox/common --stream",
```

Ditto here, package name